java/client/src/test/java/glide/managers/CommandManagerTest.java

```diff
@@ -120,22 +122,20 @@ public void submitNewCommand_return_String_result() {
     public void prepare_request_with_simple_routes(SimpleRoute routeType) {
         CompletableFuture<Response> future = new CompletableFuture<>();
         when(channelHandler.write(any(), anyBoolean())).thenReturn(future);
-        var command =
-                Command.builder().requestType(Command.RequestType.CUSTOM_COMMAND).route(routeType).build();
 
         ArgumentCaptor<RedisRequest.Builder> captor =
                 ArgumentCaptor.forClass(RedisRequest.Builder.class);
 
+        service.submitNewCommand(CustomCommand, new String[0], routeType, r -> null);
```

Comment:
why the change in order of calls?

Reply:
I moved the Mapping down to be closer to where it's being used.